README.md

```diff
@@ -20,11 +20,53 @@ And then execute:
 
 Or install it yourself as:
 
-    $ gem install volabit
+    $ git clone https://github.com/coincovemx/api-client.git
+    $ cd api-client
+    $ bundle
+    $ gem build volabit.gemspec
+    $ gem install volabit-[version].gem
 
 ## Usage
 
-TODO: Write usage instructions here
+Instance a new Volabit client object.
+
+```ruby
+app_id      = 'The registered API for your APP.'
+secret      = 'The registered secret for your APP.'
+callback    = 'The registered callback URL for your APP'
+environment = 'The environment to work on (production or test)'
+
+client = Volabit::Client.new(app_id: app_id, app_secret: secret, app_url: callback, environment: environment)
```

Comment:
I think this way is too much data... could it be like:
`Volabit::Client.new(app_id, secret, callback, environment)` ?